scripts/Doxyfile.in

```diff
@@ -1105,7 +1105,7 @@ GENERATE_HTML          = YES
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            = _api
+HTML_OUTPUT            = api/@PROJECT_VERSION@
```

Comment:
what is the motivation for changing the output directory?

Reply:
it was just an idea to create directly also the directory by Doxygen but anyway it doesn't work with recursive subdirectory creation so I need to revert this